examples/js/controls/PanoramicControls.js

```diff
@@ -0,0 +1,229 @@
+THREE.PanoramicControls = function( camera, domElement )
+{
+	this.camera = ( camera !== undefined ) ? camera : console.error( "PanoramicControls required a THREE.PerspectiveCamera in first argument." );
+	this.domElement = ( domElement !== undefined ) ? domElement : document.body;
+	this.target = new THREE.Vector3();
+
+	this.autoRotateSpeed = 1.0;
+
+	this.minPolarAngle = 0.0;// degrees
+	this.maxPolarAngle = 180.0;// degrees
+
+	this.deceleration = 0.88;
+	this.steps = 6.0;
```

Comment:
`steps` is a mouse-sensitivity parameter in your code. See how this is done in the other controls. Perhaps there is a better name for this...